### DISCUSSION TITLE
Integration with Selenium Base

### DISCUSSION THREAD

**Discussion by AlexPaiva** [kaliiiiiiiiii/Selenium-Profiles#116] [Ideas]
Hey,

Any ideas on integration with [selenium base](https://github.com/seleniumbase/SeleniumBase)? It's lacking a anti-fingerprint technique and your profiling is perfect for that, what do you think?

**Comment by ttlns** [comment]
mhh selenium-profiles is detectable by design. SeleniumBase won't change anything on that.
Also, we'd ned to rebuild thr package based on seleniumbase.
Don't think there's an easy way to integrate it.

**Reply by AlexPaiva** [reply]
Ah gotcha, was thinking the profiles could be used to generate randomized fingerprints.
Any plans for something like that open source?

**Reply by ttlns** [reply]
Well I do plan to make driverless allow emulation, however, this won't be done in near future. Lot of work needed:)
